package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Dec 10 08:45:09 UTC 2020 - José Iván López González <jlopez@suse.com>
+
+- Partitioner: do not show summary when there are no actions to
```

Comment:
NP: Actually the fix not only stop displaying the summary when it is not useful. It _stop considering the system edited_ if there are no actions to perform. Although I understand that the bug report is only about the summary.

Reply:
Yes, it has more implications (next/finish button, abort message, etc), but I did not consider relevant to list all them.